### ISSUE TITLE
No sound in local or scheduledLocal notification ( Android )

### ISSUE THREAD

**Issue by Return-1** [zo0r/react-native-push-notification#1143]
Using the latest version, local notifications yield no sound in any devices used.

**Comment by LuisBonsembiante** [comment]
The same to me!, also with vibration

**Comment by Return-1** [comment]
@LuisBonsembiante vibration works for me if sound is disabled on the phone but local notifications not yielding any sound is kind of problematic as the users wont pay any attention.

**Comment by LuisBonsembiante** [comment]
@Return-1  yes you are right about the vibrate.... I debbug de library and the line that return the sound location give me: "content://settings/system/notification_sound"..but  i try to found this route in the Xiami note 6 and dont have lucky... may be that is the problem or I wrong with my found.
